opnode/rollup/sync/reference.go

```diff
@@ -26,10 +26,10 @@ type L2Client interface {
 
 // ChainSource provides access to the L1 and L2 block graph
 type ChainSource interface {
-	L1NodeByNumber(ctx context.Context, l1Num uint64) (eth.L1Node, error)
-	L1HeadNode(ctx context.Context) (eth.L1Node, error)
-	L2NodeByNumber(ctx context.Context, l2Num *big.Int) (eth.L2Node, error)
-	L2NodeByHash(ctx context.Context, l2Hash common.Hash) (eth.L2Node, error)
+	L1NodeByNumber(ctx context.Context, l1Num uint64) (eth.L1BlockRef, error)
+	L1HeadNode(ctx context.Context) (eth.L1BlockRef, error)
+	L2NodeByNumber(ctx context.Context, l2Num *big.Int) (eth.L2BlockRef, error)
+	L2NodeByHash(ctx context.Context, l2Hash common.Hash) (eth.L2BlockRef, error)
```

Comment:
The single PR would be nice. It would have been a pain to do the rename before this PR though, so doing it after requires people to review with the slightly confusing names.

Reply:
Also, I updated the names